src/main/scala/consul/v1/kv/KvRequests.scala

```diff
@@ -16,7 +16,9 @@ trait KvRequests {
 
 object KvRequests {
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): KvRequests = new KvRequests{
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): KvRequests = new KvRequests{
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759092/issues/source?bid=2788492&fileBranchId=2939115#l21)